ci/README.md

```diff
@@ -1,13 +1,33 @@
-# Developer use of CI Docker image
+# Developer use of CI Docker images
 
-The Docker image at [`lyft/envoy-build:<hash>`](https://hub.docker.com/r/lyft/envoy-build/) is used for Travis CI checks, where `<hash>` is specified in
-[ci_steps.sh](https://github.com/lyft/envoy/blob/master/ci/ci_steps.sh). Developers
-may work with `lyft/envoy-build:latest` to provide a self-contained environment for
-building Envoy binaries and running tests that reflects the latest built image. Moreover, the Docker
-image at [`lyft/envoy:<hash>`](https://hub.docker.com/r/lyft/envoy/) is an image that has an Envoy binary at `usr/local/bin/envoy`. The `<hash>`
-corresponds to the master commit at which the binary was compiled. Lastly, `lyft/envoy:latest` contains
-an Envoy binary built from the latest tip of master that passed tests.
+Two flavors of Envoy Docker images, based on Ubuntu and Alpine Linux, are built.
 
+## Ubuntu envoy image
+The Ubuntu based Envoy Docker image at [`lyft/envoy-build:<hash>`](https://hub.docker.com/r/lyft/envoy-build/) is used for Travis CI checks,
+where `<hash>` is specified in [ci_steps.sh](https://github.com/lyft/envoy/blob/master/ci/ci_steps.sh). Developers
+may work with `lyft/envoy-build:latest` to provide a self-contained environment for building Envoy binaries and
+running tests that reflects the latest built Ubuntu Envoy image. Moreover, the Docker image
+at [`lyft/envoy:<hash>`](https://hub.docker.com/r/lyft/envoy/) is an image that has an Envoy binary at `/usr/local/bin/envoy`. The `<hash>`
+corresponds to the master commit at which the binary was compiled. Lastly, `lyft/envoy:latest` contains an Envoy
+binary built from the latest tip of master that passed tests.
+
+## Alpine envoy image
+
+Minimal images based on alpine Linux allow for quicker deployment of Envoy. Two alpine based images are built,
+one with an Envoy binary with debug symbols and one stripped of them (and consequently the smallest of the built images).
+
+The Alpine based Envoy Docker image at [`lyft/envoy-alpine:<hash>`](https://hub.docker.com/r/lyft/envoy-alpine/), and the
+Alpine based debug Envoy Docker image at [`lyft/envoy-alpine-debug:<hash>`](https://hub.docker.com/r/lyft/envoy-alpine-debug/) are
+both used for Travis CI checks, where `<hash>` is specified in [ci_steps.sh](https://github.com/lyft/envoy/blob/master/ci/ci_steps.sh).
+
+As with the Ubuntu based images, for the Alpine based images, developers
+may work with `lyft/envoy-alpine:latest` and `lyft/envoy-alpine-debug:latest` to provide a self-contained environment for
+building Envoy binaries and running tests that reflects the latest built Alpine Envoy image.
```

Comment:
This is incorrect, the images don't have an environment to build envoy, they just have an envoy binary in them. Again, if you take the edit from above you can just delete this paragraph.